### DISCUSSION TITLE
SCons blocked by AntiMalware

### DISCUSSION THREAD

**Discussion by Tsar06** [SCons/scons#4296] [General]
FYI guys, I am working with one global company...
Seems like we can't get around rejection of the downloaded zip from antimalware software.

Proxy: Anti-Malware Engine (Internal Error)
An engine error has occurred that prevents filtering and your administrator doesn't allow to content without being filtered. Please call your administrator with the error message below.

More information:

Error.Message: (14002) Internal antivirus filter error.
URL:https://deac-ams.dl.sourceforge.net/project/scons/scons/4.4.0/SCons-4.4.0.zip
URL.Host:deac-ams.dl.sourceforge.net (185.34.27.55)
Block.Reason: Proxy: Anti-Malware Engine (Internal Error) (1)
Rules.CurrentRule.Name: Anti-Malware: Standard Setting for Trusted Sites
Rules.CurrentRuleSet.Name: Gateway Anti-Malware

Also installing using pip will return error 403 including --trusted-host

and...

git clone https://github.com/SCons/scons will give SSL certificate problem as well.

Cloning into 'scons'...
fatal: unable to access 'https://github.com/SCons/scons/': SSL certificate problem: unable to get local issuer certificate--trusted-host

I have tried using my own personal PC with no McAfee malware, the download works.

Thanks

**Comment by Tsar06** [comment]
Trying accessing the wheel directly...

https://files.pythonhosted.org/packages/1a/e1/7fad50e1baa7068e3e2382b11c4d00190238798d62bfa6bbfa8619c19f39/SCons-4.4.0-py3-none-any.whl

gives the following:

Proxy: Anti-Malware Engine (Internal Error)
An engine error has occurred that prevents filtering and your administrator doesn't allow to content without being filtered. Please call your administrator with the error message below.

More information:

Error.Message: (14002) Internal antivirus filter error.
URL:https://files.pythonhosted.org/packages/1a/e1/7fad50e1baa7068e3e2382b11c4d00190238798d62bfa6bbfa8619c19f39/SCons-4.4.0-py3-none-any.whl
URL.Host:files.pythonhosted.org (151.101.129.63)
Block.Reason: Proxy: Anti-Malware Engine (Internal Error) (1)
Rules.CurrentRule.Name: Anti-Malware: Standard Setting for Trusted Sites
Rules.CurrentRuleSet.Name: Gateway Anti-Malware

**Comment by mwichmann** [comment]
Hard to know what to say here.  The error doesn't seem actionable, as I read it at least, it's not actually reporting a problem with SCons?

SCons doesn't have any binary bits, being pure Python, so the usual "virus/malware signature in the database accidentally hit something in object code" doesn't tend to trip.

**Reply by Tsar06** [reply]
Thank you Mats for responsivness. Apparently our McAfee admin resolved the problem... fixing the malware filter. So on our side I'm afraid. 
Subject resolved.

**Comment by bdbaddog** [comment]
Next time, please shoot a message to the users mailing list, or message us on our Discord server.
